static/css/slick.css

```diff
@@ -4,7 +4,7 @@
 
 .slick-list { position: relative; overflow: hidden; display: block; margin: 0; padding: 0; }
 .slick-list:focus { outline: none; }
-.slick-loading .slick-list { background: #fff url("./ajax-loader.gif") center center no-repeat; }
+.slick-loading .slick-list { background: #fff url("/static/img/ajax-loader.gif") center center no-repeat; }
```

Comment:
I agree but I'm not sure that we can set a custom loader. I think it might be hard-coded to the slick js source because it doesn't seem to be [documented anywhere](https://github.com/kenwheeler/slick/)